test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -230,40 +232,73 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 
 	When("pod runs with the CPU load balancing runtime class", func() {
 		var smtLevel int
+		var allTestpods map[types.UID]*corev1.Pod
 		var testpod *corev1.Pod
 
 		// getCPUswithLoadBalanceDisabled Return cpus which are not in any scheduling domain
 		getCPUswithLoadBalanceDisabled := func() ([]string, error) {
 			cmd := []string{"/bin/bash", "-c", "cat /proc/schedstat"}
-			schedstat, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+			schedstatData, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
```

Comment:
Maybe if we are changing this area already we can consider extarcting `getCPUswithLoadBalanceDisabled` function for better readability if an error occurs. Not a must.

Reply:
good idea, on it